docs/source/03_tutorial/07_set_up_experiment_tracking.md

```diff
@@ -0,0 +1,156 @@
+    score = r2_score(y_test, y_pred)
+    mae = mean_absolute_error(y_test, y_pred)
+    me = max_error(y_test, y_pred)
+    logger = logging.getLogger(__name__)
+    logger.info("Model has a coefficient R^2 of %.3f on test data.", score)
+    return {"r2_score": score, "mae": mae, "max_error": me}
+```
+
+The next step is to ensure that the dataset is also specified as an output of your `evaluate_model` node. Under `/pipelines/data_processing/pipeline.py`, specify the `output` of your `evaluate_model` to be the `metrics` dataset. Note that it is crucial that the output dataset exactly matches the name of the tracking dataset specified in the catalog file.
+
+The node of the `evaluate_model` on the pipeline should look like this:
+
+```python
+node(
+    func=evaluate_model,
+    inputs=["regressor", "X_test", "y_test"],
+    name="evaluate_model_node",
+    outputs="metrics"
+)
+```
+
+You have to repeat the same steps for setting up the `companies_column` dataset. For this dataset you should log the column that contains the list of companies as outlined in `companies.csv` under `/data/01_raw`. Modify the `preprocess_companies` function under the `data_processing` pipeline to return the data under a key value pair, as shown below:
+
+```python
+def preprocess_companies(companies: pd.DataFrame) -> pd.DataFrame:
+    """Preprocesses the data for companies.
+
+    Args:
+        companies: Raw data.
+    Returns:
+        Preprocessed data, with `company_rating` converted to a float and
+        `iata_approved` converted to boolean.
+    """
+    companies["iata_approved"] = _is_true(companies["iata_approved"])
+    companies["company_rating"] = _parse_percentage(companies["company_rating"])
+    return companies, {"columns": companies.columns.tolist(), "data_type": "companies"}
+```
+
+Again, you will need to ensure that the dataset is also specified as an output on `pipeline.py` under the `data_processing` pipeline, as follows:
+
+```python
+node(
+    func=preprocess_companies,
+    inputs="companies",
+    outputs=["preprocessed_companies", "companies_columns"],
+    name="preprocess_companies_node",
+)
+```
+
+Having set up both datasets, you are now ready to generate your first set of experiment tracking data!
+
+## Generating Run data
+
+One of the beauty of native experiment tracking in Kedro is that all tracked data are generated and stored each time you do a Kedro run. Hence, to generat the data, simply do:
+
+```bash
+Kedro run
+```
+
+After the run completes, under `data/09_tracking`, you will now see 2 folders, `companies_column.json` and `metrics.json`. On performing a pipeline run after setting up the tracking datasets, Kedro will generate a folder under the dataset name for each tracked dataset, with each folder containing another folder named under the timestamp of the run. Each subsequent pipeline run will generate a new folder named after the timestamp of the run, in the same directory of the tracked dataset.
```

Comment:
```suggestion
After the run completes, under `data/09_tracking`, you will now see two folders, `companies_column.json` and `metrics.json`. On performing a pipeline run after setting up the tracking datasets, Kedro will generate a folder under the dataset name for each tracked dataset, with each folder containing another folder named under the timestamp of the run. Each subsequent pipeline run will generate a new folder named after the timestamp of the run, in the same directory of the tracked dataset.
```

Is this paragraph correct like this? I got a bit confused reading it 🤔 Are the last two sentences saying the same but in a different way?

Reply:
@MerelTheisenQB indeed the current sentences does sound slightly confusing - here I am trying to explain the folder structure and how new run folders gets generated on each new run.

I've updated the paragraph with new wording - let me know if the new wording sounds more clear?

Reply:
Yes it makes more sense now 👍